formtron-schema.json

```diff
@@ -47,6 +47,15 @@
           "type": "object",
           "additionalProperties": { "$ref": "#/definitions/field" }
         },
+        "layouts": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "array",
+            "items": {
+              "type":  "string"
```

Comment:
Do we like free strings or would we prefer a layout matrix

```
const strings = {
  layout1: [
    'path info host', 
    'method method method'
  ],
};

const matrix = {
  layout2: [
    ['path', 'info', 'host'],
    ['method', 'method', 'method']
  ],
};
```

Reply:
( this comment is more an open ended question). Matrix is nice because it's easer to find out which item is in which position programatically `layout2[1][2]`

Reply:
It's a tough call. Ultimately, I think free strings are better because they're quicker to type, and I think that ease of use makes it "more fun" to write layouts.

plus, it's really easy to convert them to a matrix programmatically. `rows.map(x => x.split(/\s+/)`. Ultimately I chose this format because
a) you can't do multiline strings in JSON 😢 
b) it's got the least punctuation

Reply:
Strings ARE nice too look at and agreed readability and "fun" factor are better with string lol. The programatic use of the matrix is probably rare so I'm cool with keeping strings